plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -553,10 +553,8 @@ public void testInsertToTableWithHiddenId()
     @Override
     public void testCreateTableWithLongColumnName()
     {
-        // TODO: Find the maximum column name length in Cassandra and enable this test.
         assertThatThrownBy(super::testCreateTableWithLongColumnName)
-                .hasMessageMatching(".* Mutation of .* bytes is too large.*");
-        throw new SkipException("TODO");
+                .hasMessageMatching(".*Attempted serializing to buffer exceeded maximum of 65535 bytes:.*");
```

Comment:
We shouldn't override this method anymore. It should be handled in `verifyColumnNameLengthFailurePermissible ` method.

Reply:
Addressed! Removed the override method.